docs/docs/APIdocs/layers/optim/Adadelta.md

```diff
@@ -0,0 +1,34 @@
+## Adadelta ##
+
+
+*AdaDelta* implementation for *SGD* 
+It has been proposed in `ADADELTA: An Adaptive Learning Rate Method`.
+http://arxiv.org/abs/1212.5701.
+
+**Scala:**
+```scala
+Adadelta(decayRate: Double = 0.9, Epsilon: Double = 1e-10)
+```
+**Python:**
+```python
+AdaDelta(decayrate = 0.9, epsilon = 1e-10, bigdl_type="float")
+```
+
+
+**Scala example:**
+```scala
+optimizer.setOptimMethod(new Adadelta(0.9, 1e-10))
```

Comment:
Maybe too simple? You may take a look at https://github.com/intel-analytics/BigDL/blob/master/docs/docs/APIdocs/layers/optim/Adagrad.md